espresso-server/app/src/androidTest/java/io/appium/espressoserver/EspressoServerRunnerTest.java

```diff
@@ -21,9 +21,9 @@
 public class EspressoServerRunnerTest {
 
     @Test
-    public void startEspressoServer() throws InterruptedException, IOException, ServerErrorException {
+    public void startEspressoServer() throws InterruptedException, IOException, DuplicateRouteException {
         new Server();
         // TODO: Figure out how to keep Runner open forever
-        Thread.sleep(300000);
+        Thread.sleep(3000000);
```

Comment:
I made a separate issue for that. I don't think this will work because it's a Test APK that's being generated and there's no Activity or Context.